pom.xml

```diff
@@ -276,6 +276,18 @@
 					<artifactId>findbugs-maven-plugin</artifactId>
 					<version>3.0.4</version>
 				</plugin>
+				<plugin>
```

Comment:
@kapilkalra04 We already have the maven jacoco plugin. Is it not working? If cobertura has better support with codecov, we can always use cobert

Reply:
If we go with cobertura, we can remove jacoco-maven-plugin. Since we are going to use codecov, we can also remove coveralls-maven-plugin

Reply:
The maven jacoco plugin gives an error as the project doesn't contain Python coveragepy ( [see here](https://travis-ci.org/kapilkalra04/BITS-Darshini/builds/326883907) , line 1436) which is similar to [this](https://github.com/codecov/codecov-bash/issues/94), therefore the issue boiled down to the fact that the codecov bash uploader couldn't find the coverage.xml for uploading. This is why I went with the other option. 

The jacoco plugin was already present in pom.xml file before I forked the dev branch. So do I still remove it?